outputs.tf

```diff
@@ -29,20 +29,17 @@ output "uri" {
   description = "The URI of the Pub/Sub topic"
 }
 
-output "subscription_names" {
-  value = "${concat(
-    google_pubsub_subscription.push_subscriptions.*.name,
-    google_pubsub_subscription.pull_subscriptions.*.name
+output "subscriptions" {
+  value = "${merge(
+    zipmap(
+      google_pubsub_subscription.push_subscriptions.*.name,
+      google_pubsub_subscription.push_subscriptions.*.path
+    ),
+    zipmap(
+      google_pubsub_subscription.pull_subscriptions.*.name,
+      google_pubsub_subscription.pull_subscriptions.*.path
+    )
   )}"
 
   description = "The name list of Pub/Sub subscriptions"
```

Comment:
The description must be updated to accurately reflect the contents of the value.

Reply:
Done.